integration/dev_test.go

```diff
@@ -315,21 +315,22 @@ func assertResponseFromPort(t *testing.T, port int, expected string) {
 		case <-ctx.Done():
 			t.Fatalf("Timed out waiting for response from port %d", port)
 		case <-time.After(1 * time.Second):
-			resp, err := http.Get(fmt.Sprintf("http://%s:%d", util.Loopback, port))
+			client := http.Client{Timeout: 1 * time.Second}
```

Comment:
this is the fix - I expect that sometimes the client would just hang in broken connections. This helps get over that issue and forces a retry.